### PR TITLE
3.3 Corrected spelling and grammar. 

### PR DESCRIPTION
**Pull request by paniash**
# Changes made

Changed 'indexes' to 'indices'. Added a period at the end of a sentence.
Removed code line `%config InlineBackend.figure_format = 'svg'` for brevity.

### CODE REVIEW

**Comment by paniash** [comment]
@frankharkins As mentioned in issue #710, the changes can be made to better suit beginners. Since, the table is not rendered through HTML, could you point me to how the table is rendered through an image file. I'm willing to make the required changes.

File: content/ch-algorithms/superdense-coding.ipynb [hunk 1]
```diff
@@ -142,15 +142,14 @@
     "# Importing everything\n",
     "from qiskit import *\n",
     "from qiskit.visualization import plot_histogram\n",
-    "# For Jupyter Notebooks, change the settings to get nicer images\n",
-    "%config InlineBackend.figure_format = 'svg'"
+    "# For Jupyter Notebooks, change the settings to get nicer images"
```
Comment (frankharkins) [inline content/ch-algorithms/superdense-coding.ipynb:145]: If removing the `%config` line, we should also remove this comment.

Comment (paniash) [reply content/ch-algorithms/superdense-coding.ipynb]: My apologies @frankharkins. I forgot about the comment.

**Comment by frankharkins** [comment]
@paniash thanks for offering to re-render the tables. If you open the notebook in Jupyter Notebook and edit the markdown cell containing the table image, you should see a commented-out markdown table. If you uncomment this table it will render in the notebook and you can screen-shot it, crop and save as PNG, and overwrite the old table image.

**Review by frankharkins — APPROVED** [review]

**Comment by paniash** [comment]
@frankharkins I've encountered a slight issue. On cropping the table and saving as a png file, the image size is considerably smaller than another table image. I did try to scale the image but that results in a loss of image quality. Could you suggest as to how I could scale the image to be as lossless as possible?

**Comment by paniash** [comment]
@frankharkins I've updated the table, can you please take a look and let me know if it's fine?

**Comment by paniash** [comment]
@frankharkins I've updated the table. Could you check and let me know if it's fine?

**Review by frankharkins — APPROVED** [review]

**Comment by frankharkins** [comment]
This looks good! thank you

**Pull request merged**